### PR TITLE
[12.0][IMP] - Add menu for contract tag

### PR DESCRIPTION
**Pull request by sbejaoui**

### CODE REVIEW

File: contract/views/contract_tag.xml [hunk 1]
```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2020 ACSONE SA/NV
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+
+<odoo>
+
+    <record model="ir.ui.view" id="contract_tag_form_view">
```
Comment (astirpe) [inline contract/views/contract_tag.xml]: Place id attribute before model (see [guidelines](https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#xml-files))

**Review by astirpe — APPROVED** [review]
LGTM. Only non-blocking suggestion by guidelines.

**Review by dreispt — APPROVED** [review]

**Review by bodedra — APPROVED** [review]

**Review by carlosdauden — APPROVED** [review]
Review @astirpe comment

File: contract/views/contract_tag.xml (hunk 1 above)
Comment (carlosdauden) [inline contract/views/contract_tag.xml]:
```suggestion
    <record id="contract_tag_form_view" model="ir.ui.view">
```

**Pull request merged**